index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
+
+  <body>
+<section id ="welcome-section">
+  <h1> Texto </h1>
+</section>
+<section id ="projects">
+  <h2 class="project-tile"> Projects</h2>
+  <a href="www.google.com"> link</a>
```

Comment:
🙏🏼 It is a good practice to remove spaces that do not have a specific function in our code. If possible, I recommend adjusting the text to the beginning and end of the tags that contain it.

```suggestion
  <a href="www.google.com">Link</a>
```